extension/src/experiments/data/index.ts

```diff
@@ -157,9 +156,6 @@ export class ExperimentsData extends BaseData<ExperimentsOutput> {
       })
     } catch {
       this.notifyChanged(defaultData)
-      void Toast.showError(
```

Comment:
We could add some logic to check if the user has already received the Toast and has a self-hosted URL, but that would add some extra complexity that I don't think is needed since we probably have few users using self-hosted URLs.

Reply:
We can have a health status in the Setup screen instead I think. If connection is failing we can a message, or even change it to "yellow" in the Setup screen. WDYT?